.gitignore

```diff
@@ -22,3 +22,30 @@ glove2word2vec.py
 
 .idea/
 venv/
+
+# General
+.DS_Store
```

Comment:
Can you remove changes to your gitignore? They seem rather specific to your setup.

Reply:
Ok I can remove it.

Reply:
BTW You can also create a global gitignore: https://docs.github.com/en/github/getting-started-with-github/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
@juharris Thank you for your advice. It is what I need.